pkg/services/app/app.go

```diff
@@ -77,25 +77,25 @@ func (a *App) getDeploymentType(ctx context.Context, name string, namespace stri
 	return wego.DeploymentType(app.Spec.DeploymentType), nil
 }
 
-func (a *App) getSuspendedStatus(ctx context.Context, name, namespace string, deploymentType wego.DeploymentType) (bool, error) {
-	var automation client.Object
+func (a *AppSvc) getSuspendedStatus(ctx context.Context, name, namespace string, deploymentType wego.DeploymentType) (bool, error) {
+	var automationClient client.Object
```

Comment:
I had to change the name because it conflicted with the `automation` package and landed on`automationClient` since it was a `client.Object`. I'm happy to change it.